docs/markdown/Reference-manual.md

```diff
@@ -2000,6 +2000,16 @@ Dictionaries are available since 0.47.0.
 Since 0.48.0 dictionaries can be added (e.g. `d1 = d2 + d3` and `d1 += d2`).
 Values from the second dictionary overrides values from the first.
 
+### Feature option object
```

Comment:
This should be moved in the `## Returned objects` section just below.

Reply:
also, there should be a link to this from the text in `get_option()` above which mentions these objects.

Reply:
Both done.